analysis_options.yaml

```diff
@@ -1 +1 @@
-include: package:lint/analysis_options_package.yaml
+
```

Comment:
I think that is important to leave a comment explaining why the include is set here. Probably adding the link to the workaround as well. WDYT?

Reply:
Done!